app/views/brexit_checker/save_results.html.erb

```diff
@@ -0,0 +1,55 @@
+<% content_for :title, t('brexit_checker.account_signup.title') %>
+<% content_for :head do %>
+  <% unless params[:page] %>
+    <meta name="robots" content="noindex">
+  <% end %>
+<% end %>
+
+<div class="govuk-width-container">
+  <%= render 'govuk_publishing_components/components/breadcrumbs', {
+    collapse_on_mobile: true,
+    breadcrumbs: [
+      {
+        title: t('brexit_checker.breadcrumbs.home'),
+        url: "/"
+      },
+      {
+        title: t('brexit_checker.breadcrumbs.brexit-home'),
+        url: "/transition"
+      },
+      {
+        title: t('brexit_checker.breadcrumbs.results'),
+        url: transition_checker_results_path(c: criteria_keys)
+      }
+    ]
+  } %>
+
+  <main class="govuk-main-wrapper" role="mail">
```

Comment:
I've not heard of a `role="mail"` before, and I can't find anything online about it. Where did this come from?

Reply:
:thinking: Humm so that's some copy-pasta, from [here](https://github.com/alphagov/finder-frontend/blob/master/app/views/brexit_checker/email_signup.html.erb#L27).

Which seems to have been introduced [here](https://github.com/alphagov/finder-frontend/commit/49f5bc1b93030e0ec4dc3aaaef33f3b53298a851).

:thinking: I can't see that [anywhere here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) so i'm not sure about the origin, will remove from my markup, do you think it should be removed from the other place too?

Reply:
Removed from [here](https://github.com/alphagov/finder-frontend/pull/2207/commits/75c88dbe150c203702e3e0027a0d7c6346011867#diff-b86895025bcaff899c12bc2663d315b2R27)